### PR TITLE
Improve Swedish translations

### PR DESCRIPTION
**Pull request by ErikXXon**
Continue working on spelling mistakes and wording in the translation

### CODE REVIEW

**Comment by ykzts** [comment]
Please resolve conflicts or use [Weblate](https://weblate.joinmastodon.org/).

File: app/javascript/mastodon/locales/sv.json [hunk 1]
```diff
@@ -11,21 +11,21 @@
   "account.follows_you": "Följer dig",
   "account.hide_reblogs": "Dölj knuffar från @{name}",
   "account.media": "Media",
-  "account.mention": "Nämna @{name}",
+  "account.mention": "Omnämn @{name}",
+  "account.mute": "Tysta ner @{name}",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/sv.json]: alphabetical

File: app/javascript/mastodon/locales/sv.json [hunk 2]
```diff
@@ -54,8 +54,8 @@
   "column_header.unpin": "Ångra fäst",
   "column_subheading.navigation": "Navigation",
   "column_subheading.settings": "Inställningar",
+  "compose_form.lock_disclaimer": "Ditt konto är inte {locked}. Vemsomhelst kan följa dig och även se dina inlägg som är skrivna för endast dina följare.",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/sv.json]: alphabetical

File: app/javascript/mastodon/locales/sv.json [hunk 3]
```diff
@@ -181,8 +181,8 @@
   "onboarding.page_four.home": "Hemmatidslinjen visar inlägg från personer du följer.",
   "onboarding.page_four.notifications": "Meddelandekolumnen visar när någon interagerar med dig.",
   "onboarding.page_one.federation": "Mastodon är ett nätverk av oberoende servrar som ansluter för att skapa ett större socialt nätverk. Vi kallar dessa servrar instanser.",
+  "onboarding.page_one.handle": "Du är på {domain}, så din fulla hantering är {handle}",
   "onboarding.page_one.full_handle": "Ditt fullständiga användarnamn/mastodonadress",
-  "onboarding.page_one.handle_hint": "Det här är vad du skulle berätta för dina vänner att söka efter.",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/sv.json]: Why did you delete this line?

File: app/javascript/mastodon/locales/sv.json [hunk 4]
```diff
@@ -181,8 +181,8 @@
   "onboarding.page_four.home": "Hemmatidslinjen visar inlägg från personer du följer.",
   "onboarding.page_four.notifications": "Meddelandekolumnen visar när någon interagerar med dig.",
   "onboarding.page_one.federation": "Mastodon är ett nätverk av oberoende servrar som ansluter för att skapa ett större socialt nätverk. Vi kallar dessa servrar instanser.",
+  "onboarding.page_one.handle": "Du är på {domain}, så din fulla hantering är {handle}",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/sv.json]: why?

File: app/javascript/mastodon/locales/sv.json [hunk 5]
```diff
@@ -259,6 +259,7 @@
   "status.show_more_all": "Show more for all",
   "status.unmute_conversation": "Öppna konversation",
   "status.unpin": "Ångra fäst i profil",
+  "tabs_bar.compose": "Skriv",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/sv.json]: why?

**Review by ykzts — CHANGES_REQUESTED** [review]

File: app/javascript/mastodon/locales/sv.json (hunk 3 above)
Comment (ErikXXon) [reply app/javascript/mastodon/locales/sv.json]: mistake if that line got deleted :(

File: app/javascript/mastodon/locales/sv.json (hunk 4 above)
Comment (ErikXXon) [reply app/javascript/mastodon/locales/sv.json]: Formulate

File: app/javascript/mastodon/locales/sv.json (hunk 5 above)
Comment (ErikXXon) [reply app/javascript/mastodon/locales/sv.json]: Formulate

File: app/javascript/mastodon/locales/sv.json (hunk 4 above)
Comment (ykzts) [reply app/javascript/mastodon/locales/sv.json]: has been deleted.

File: app/javascript/mastodon/locales/sv.json (hunk 5 above)
Comment (ykzts) [reply app/javascript/mastodon/locales/sv.json]: has been deleted
